resources/files/puppet-agent.sh

```diff
@@ -1,11 +1,11 @@
 # Add /opt/puppetlabs/bin to the path for sh compatible users
 
-if [ -z ${PATH-} ] ; then
+if [ -z "${PATH-}" ] ; then
   export PATH=/opt/puppetlabs/bin
-elif ! echo ${PATH} | grep -q /opt/puppetlabs/bin ; then
-  export PATH=${PATH}:/opt/puppetlabs/bin
+elif ! echo "${PATH}" | grep -q /opt/puppetlabs/bin ; then
+  export PATH="${PATH}":/opt/puppetlabs/bin
```

Comment:
I'd just enclose the entire value in quotes, like: `PATH="${PATH}:/opt/puppetlabs/bin"`, same for `MANPATH`. Works the same but in my opinion it reads better this way.

Reply:
Indeed looks better as you proposed! Fixed!